### DISCUSSION TITLE
access to Azure without credit

### DISCUSSION THREAD

**Discussion by Reshma-kumari-GitHub** [github/india#362] [Students]
Can anyone please help me in getting access to azure without providing credit card details? It would be a great help.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏
